core/src/main/java/bio/terra/pearl/core/service/datarepo/DataRepoExportService.java

```diff
@@ -205,20 +213,23 @@ public void pollRunningJobs() {
     private void pollAndUpdateCreateJobStatus(DataRepoJob job) {
         try {
             JobStatusEnum jobStatus = dataRepoClient.getJobStatus(job.getTdrJobId()).getJobStatus();
+            Dataset existingDataset = datasetDao.findByDatasetName(job.getDatasetName()).get();
```

Comment:
this can be removed with https://github.com/broadinstitute/pearl/pull/386/files#r1198205721